portal/views/tou.py

```diff
@@ -94,7 +94,9 @@ def get_tou(user_id):
         abort(404)
     current_user().check_role(permission='view', other_id=user_id)
 
-    tous = ToU.query.join(Audit).filter(Audit.user_id == user_id)
+    tous = ToU.query.join(Audit).filter(
+        and_(Audit.user_id == user_id,
```

Comment:
Again, I think we want `subject_id`, no?

Reply:
(see similar comment above)